config.yaml

```diff
@@ -112,3 +112,34 @@ languages:
           url: https://github.com/go-gitea/
           weight: 60
           pre: github
+
+  pt-br:
+    weight: 3
+    languageName: Português brasileiro
```

Comment:
@andreynering but Brazilian portuguese and worldwide portuguese have significant differences and if put only "Português", will not to be possible to people from Portugual create pt-pt translation because both will have name "Português".